### PR TITLE
[blog-sample-data] Changes on banner text on the Homepage

### PR DESCRIPTION
**Pull request by chmst**
### Summary of Changes
A new article for typogrphy has been added to blog sample data. 
The call-to-action button "Learn more" leads nowhere until now, it now goes to typography
The link to typography also has been added to the blog sample menu. 

### Testing Instructions

Install the blog sample data.

### Expected result
A Linkt to typography in the top menu,
A link to the article typography on the banner area. 
The article typography shows - depending on the chosen fonts, a collection of headings colours and classes

### CODE REVIEW

**Comment by brianteeman** [comment]
Personal opinion I am not in favour of styling text as a button. Users think the button is broken ;)

**Comment by richard67** [comment]
We have put much effort in making colours fit together, the cyan colour breaks it. It doesn’t fit at all to the template colours.

**Comment by richard67** [comment]
I suggest to make the separator white and the button have white background and the same dark blue text colors as we have elsewhere e.g. for links.

The button should lead to some page which makes sense, and this should not be the current page like it is now without this PR in the development branch. If that is technically not possible with reasonable effort, I suggest to remove that button completely.

**Comment by richard67** [comment]
With the old text "Learn more" for the button, I would have expected to go to a page where I can learn something more about Joomla 4 or the Cassiopeia template. This could have been an internal link or an external link to docs or help pages.

With the new text "Enjoy!" I would at least expect something to happen. Ideally it would be an internal link to some page where you can start to do something, e.g. the Author or Admin login or how we called it now, or some page of the sample layouts where you can see something new or special for J4, or whatever else, and if it is just a modal being shown with a longer text "Have fun exploring Joomla 4 with the Cassiopeia template. Just check the menu at the top to find new fancy things" or whatever else. But there should be some kind of action when clicking on a button, not just nothing.

**Comment by richard67** [comment]
Hmm, I see, the colours come from our Bootstrap 4 colours for the button types, info, success, primary or whatever else we have.

Maybe we have to adjust these colours to the template colouring?

@drmenzelit What do you think? Or do you like the cyan, and it's just me thinking it doesn't fit?

**Comment by drmenzelit** [comment]
We could need a designer that adapt the colours for info, success, etc. to fit better to the template...

**Comment by chmst** [comment]
Thank you for your opinion. So at least we have started a decision process. I make this as draft. 

The colour comes from template variables - yes, it would be nice to have a designer who defines a good colour scheme for all colour variables. 

And yes, the button originally should lead to a page with more information. We have not many articles - mainly because it is a challenge for translators.

**Comment by chmst** [comment]
Reverted colour changes but added a new article typography.

File: administrator/language/en-GB/plg_sampledata_blog.ini [hunk 1]
```diff
@@ -124,8 +124,8 @@ PLG_SAMPLEDATA_BLOG_SAMPLEDATA_MODULES_MODULE_5_TITLE="Older Posts"
 PLG_SAMPLEDATA_BLOG_SAMPLEDATA_MODULES_MODULE_6_TITLE="Bottom Menu"
 PLG_SAMPLEDATA_BLOG_SAMPLEDATA_MODULES_MODULE_7_TITLE="Search"
 PLG_SAMPLEDATA_BLOG_SAMPLEDATA_MODULES_MODULE_8_TITLE="Image"
+PLG_SAMPLEDATA_BLOG_SAMPLEDATA_MODULES_MODULE_8_CONTENT="<h1 class=\"display-4 text-thin\">Welcome to Joomla on Cassiopeia!</h1><p class=\"lead\"><strong>You have chosen one of the most powerful CMS Systems in the world.</strong></p><p class=\"lead\">Cassiopeia is the multi-purpose frontend template for Joomla 4.</p><p><a class=\"btn btn-primary btn-lg\" href=\"%s\">Typography »</a></p>"
```
Comment (richard67) [inline administrator/language/en-GB/plg_sampledata_blog.ini:126]: Please alpha order the language strings, i.e. put the new sting PLG_SAMPLEDATA_BLOG_SAMPLEDATA_MODULES_MODULE_8_CONTENT before PLG_SAMPLEDATA_BLOG_SAMPLEDATA_MODULES_MODULE_8_TITLE.

**Review by richard67 — CHANGES_REQUESTED** [review]

File: administrator/language/en-GB/plg_sampledata_blog.ini [hunk 2]
```diff
@@ -12,12 +12,15 @@ PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_0_TITLE="About"
...
... sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex. (p)</p><p> </p><a href=\"link\"> Lorem ipsum dolor (a)</a></p><p><span class=\"btn btn-primary\">primary</span><span class=\"btn btn-secondary\">secondary</span><span class=\"btn btn-info\">info</span><span class=\"btn btn-success\">success</span><span class=\"btn btn-warning\">warning</span><span class=\"btn btn-danger\">danger</span></p><p><span class=\"text text-info\">text-info</span>  <span class=\"text text-danger\">text-danger</span>  <span class=\"text text-primary\">text-primary</span>  <span class=\"text text-secondary\">text-secondary</span></p><h4>(ol)(li)</h4><ol><li>Lorem ipsum dolor sit amet consectetur</li><li>Lorem ipsum dolor sit amet consectetur</li></ol><p> </p><h4>(ul)(li)</h4><ul><li>Lorem ipsum dolor sit amet consectetur</li><li>Lorem ipsum dolor sit amet consectetur</li></ul><p class=\"display-1\">Lorem (display-1)</p><p class=\"display-2\">Lorem (display-2)</p><p class=\"display-3\">Lorem ipsum (display-3)</p><p class=\"display-4\">Lorem ipsum (display-4)</p></div>"
```
Comment (richard67) [inline administrator/language/en-GB/plg_sampledata_blog.ini]:
```suggestion
PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_10_INTROTEXT="<div><h1>Typography in Cassiopeia (h1)</h1><p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex. (p)</p><h2>Lorem ipsum dolor sit amet (h2)</h2><p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex. (p)</p><h3>Lorem ipsum dolor sit amet (h3)</h3><p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex. (p)</p><h4>Lorem ipsum dolor sit amet (h4)</h4><p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex. (p)</p><h5>Lorem ipsum dolor sit amet (h5)</h5><p> </p><p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex. (p)</p><p> </p><a href=\"link\"> Lorem ipsum dolor (a)</a></p><p><span class=\"btn btn-primary\">primary</span><span class=\"btn btn-secondary\">secondary</span><span class=\"btn btn-info\">info</span><span class=\"btn btn-success\">success</span><span class=\"btn btn-warning\">warning</span><span class=\"btn btn-danger\">danger</span></p><p><span class=\"text text-info\">text-info</span>  <span class=\"text text-danger\">text-danger</span>  <span class=\"text text-primary\">text-primary</span>  <span class=\"text text-secondary\">text-secondary</span></p><h4>(ol)(li)</h4><ol><li>Lorem ipsum dolor sit amet consectetur</li><li>Lorem ipsum dolor sit amet consectetur</li></ol><p> </p><h4>(ul)(li)</h4><ul><li>Lorem ipsum dolor sit amet consectetur</li><li>Lorem ipsum dolor sit amet consectetur</li></ul><p class=\"display-1\">Lorem (display-1)</p><p class=\"display-2\">Lorem (display-2)</p><p class=\"display-3\">Lorem ipsum (display-3)</p><p class=\"display-4\">Lorem ipsum (display-4)</p></div>"
```
Add missing space before the "(h1)" to be consistent with the other "(...)".

File: administrator/language/en-GB/plg_sampledata_blog.ini (hunk 1 above)
Comment (chmst) [reply administrator/language/en-GB/plg_sampledata_blog.ini:126]: done

File: administrator/language/en-GB/plg_sampledata_blog.ini (hunk 2 above)
Comment (chmst) [reply administrator/language/en-GB/plg_sampledata_blog.ini]: thank you, done

File: administrator/language/en-GB/plg_sampledata_blog.ini (hunk 1 above)
Comment (richard67) [reply administrator/language/en-GB/plg_sampledata_blog.ini:126]: No, this point has not been done as far as I can see.

**Review by richard67 — APPROVED** [review]
Code review ok, real test will follow.

**Comment by richard67** [comment]
- Is there a warning colour for text, too, like there is for buttons? If so, it is missing. I get only `text-secondary text-primary text-info text-success text-danger` If there is a `text-warning`, too, it should be between `text-success` and `text-danger`.
- The link with text `Lorem ipsum dolor (a)` points to a not existing local URL with `/link` at the end. Couldn't we make it point to just the index.php so we don't get a 404?

**Comment by chmst** [comment]
The colour warning for text on a white background has not sufficient contrast-ratio and should not be used. I am or sure if it is good to show different text clours at all.

**Review by richard67 — APPROVED** [review]

**Comment by richard67** [comment]
I have tested this PR :white_check_mark: with success.

**Review by richard67 — APPROVED** [review]

**Comment by richard67** [comment]
I have tested this PR :white_check_mark: with success.

**Comment by richard67** [comment]
Thanks!

**Comment by chmst** [comment]
Thank you!

**Pull request merged**